base/test.jl

```diff
@@ -271,6 +276,17 @@ if an exception is thrown inside the test block but outside of a `@test` context
 function record end
 
 """
+    should_run(ts::AbstractTestSet, nruns::Int)
+
+Decide if the tests should be run; returns true if and only if they should.
+The tests are called in a loop and the `nruns` arg counts the number of times the test
+block has been executed. `nruns` is 0 when the tests have not yet executed.
+"""
+function should_run(ts::AbstractTestSet, nruns::Int)
```

Comment:
No, but it seems good to have a sensible default which preserves the previous behavior. Do you think it is better to implement this for the fallback only and leave it empty for AbstractTestSet?